.clang-format

```diff
@@ -0,0 +1,25 @@
+BasedOnStyle: LLVM
+Language: C
+IndentWidth: 2
```

Comment:
Wanted to double-check if this was a typo or intentional: I've admittedly not looked at much C code, but I don't think I've ever seen C code with only two spaces of indentation, usually it's 4 or more?

Reply:
This is intentional as I prefer to save some columns when the language doesn't mandate indentation. For example, the [Google C++ style guide](https://google.github.io/styleguide/cppguide.html#Spaces_vs._Tabs) uses 2-space indentation, some popular codebases like [glib](https://github.com/GNOME/glib) and [sqlite](https://github.com/sqlite/sqlite) also use 2-space indentation.